tests/lib/linter/linter.js

```diff
@@ -12562,7 +12562,7 @@ var a = "test2";
                     const code = `
                     /* eslint-disable-next-line no-alert --
                         description on why this exception is seen as appropriate but past a
-                        comfortable reading line length 
+                        comfortable reading line length
```

Comment:
This is also okay, it doesn't look like this test was intended to have a trailing space here, so it's better to remove it to avoid confusion.

Reply:
Yeah, I think that was just an EditorConfig change rather than an intentional one on my part